contrib/go-chi/chi/chi.go

```diff
@@ -38,20 +38,19 @@ func Middleware(opts ...Option) func(next http.Handler) http.Handler {
 		fn(cfg)
 	}
 	log.Debug("contrib/go-chi/chi: Configuring Middleware: %#v", cfg)
-	spanOpts := options.Copy(cfg.spanOpts...) // spanOpts must be a copy of cfg.spanOpts, locally scoped, to avoid races.
-	spanOpts = append(spanOpts, tracer.ServiceName(cfg.serviceName),
+	spanOpts := append(cfg.spanOpts, tracer.ServiceName(cfg.serviceName),
```

Comment:
This one too. It's not defensive, we could change user stuff.

Reply:
That's fine though, right? We talked internally about how opts passed into a `Middleware` function like this one shouldn't be used/changed by the user after they've called this method. It's not a race condition issue so much as it is an issue with possible side effects here, right?

Reply:
Discussed offline, this is fine.